app/code/Magento/CatalogUrlRewrite/Observer/AfterImportDataObserver.php

```diff
@@ -478,7 +478,7 @@ private function currentUrlRewritesRegenerate()
             ]
         );
 
-        $urlRewrites = [];
+        $urlRewritesRaw = [[]];
```

Comment:
I thing it's better to add empty array to array_merge as first argument. Also creating new variable it's really needed here
```suggestion
        $urlRewrites = [];
```